src/java/org/httpkit/HttpUtils.java

```diff
@@ -257,6 +257,14 @@ public static String getHost(URI uri) {
         }
         return host;
     }
+    
+    public static String getProxyHost(URI uri){
```

Comment:
I think this function is telling us that `:proxy "http://some.proxy:8080"` is not a good way to express the proxy.

Should we recommend changing this to `:proxy-host "some.proxy" :proxy-port 8080`